modules/installation-configuration-parameters.adoc

```diff
@@ -61,6 +61,7 @@ ifeval::["{context}" == "installing-aws-government-region"]
 endif::[]
 ifeval::["{context}" == "installing-aws-secret-region"]
 :aws:
+:aws-secret:
```

Comment:
I believe this is an artifact of your previous change. Given that there is no longer text specific to the secret region assembly, please remove `:aws-secret:`

Reply:
Done!